implementations/rust/ockam/ockam_command/src/subcommand.rs

```diff
@@ -254,28 +254,33 @@ impl OckamSubcommand {
         }
     }
 
-    /// Return the node name for an ockam node create command
-    pub fn node_name(&self) -> Option<String> {
+    /// Return true if this command represents the execution of a local node
+    pub fn is_local_node(&self) -> bool {
         match self {
             OckamSubcommand::Node(cmd) => match &cmd.subcommand {
                 NodeSubcommand::Create(cmd) => {
-                    if cmd.foreground_args.child_process {
-                        Some(cmd.name.clone())
-                    } else {
-                        None
-                    }
+                    cmd.foreground_args.foreground || cmd.foreground_args.child_process
                 }
+                _ => false,
+            },
+
+            OckamSubcommand::Authority(cmd) => match &cmd.subcommand {
+                AuthoritySubcommand::Create(cmd) => cmd.foreground || cmd.child_process,
+            },
+            _ => false,
+        }
+    }
+
+    /// Return the node name for an ockam node create command
+    pub fn node_name(&self) -> Option<String> {
```

Comment:
This function doesn't seem to be used. Should we remove it?

Reply:
Now it's used again :-).